styles/mixins.scss

```diff
@@ -0,0 +1,10 @@
+%focus-border {
```

Comment:
let's rename this extend to `%focus-border-on-after` in order to understand that `::after` will be occupied when we use this extend